open_mastr/utils/helpers.py

```diff
@@ -295,25 +297,10 @@ def transform_data_parameter(
     return data, api_data_types, api_location_types, harmonisation_log
 
 
-def transform_date_parameter(self, method, date, **kwargs):
+def transform_date_parameter(method, date, **kwargs):
     if method == "bulk":
         date = kwargs.get("bulk_date", date)
         date = "today" if date is None else date
-        if date == "existing":
```

Comment:
Here also we still need the existing stuff. Did you merge develop from open_mastr into develop at Grazvy?